var/spack/repos/builtin/packages/gtkplus/package.py

```diff
@@ -39,7 +39,6 @@ class Gtkplus(Package):
     depends_on("atk")
     depends_on("gdk-pixbuf")
     depends_on("glib")
-    depends_on("pango")
```

Comment:
If you remove this dependency, the following command:
```
spack spec gtkplus ^pango@1.36.8
```
will say that gtkplus does not depend on pango.